modules/apps/object/object-service/src/main/java/com/liferay/object/internal/item/selector/SystemObjectEntryItemSelectorView.java

```diff
@@ -226,11 +252,22 @@ public String getTitle(Locale locale) {
 				return StringPool.BLANK;
 			}
 
-			Map<String, Object> modelAttributes =
-				_baseModel.getModelAttributes();
+			User user = _userLocalService.fetchUser(
+				PrincipalThreadLocal.getUserId());
+
+			Object titleFieldValue = ObjectEntryValuesUtil.getTitleFieldValue(
+				objectField.getBusinessType(), _baseModel.getModelAttributes(),
+				objectField, user,
+				ObjectEntryDTOConverterUtil.toDTO(
+					_baseModel, _dtoConverterRegistry,
+					_objectDefinition.getName(),
+					_systemObjectDefinitionManagerRegistry, user));
+
+			if (titleFieldValue == null) {
+				return StringPool.BLANK;
+			}
 
-			return GetterUtil.getString(
-				modelAttributes.get(objectField.getDBColumnName()));
+			return titleFieldValue.toString();
```

Comment:
@devpedrotavares @seltong can we use `GetterUtil.getString()` instead?

Reply:
The `titleFieldValue` variable is an Object and it will not always be a an instance of a String. It could be a Boolean or Date, for example. If we call `GetterUtil.getString()` passing an Object that is not a String, it will return an empty string instead of calling `toString()` which is what we want.